internal/immich/immich.go

```diff
@@ -241,3 +243,8 @@ func FlushApiCache() {
 func ApiCacheCount() int {
 	return apiCache.ItemCount()
 }
+
+func GetApiCacheKey(apiUrl string, requestConfig *config.Config) string {
+	configJson, _ := json.Marshal(requestConfig)
+	return fmt.Sprintf("%s:%s", apiUrl, configJson)
```

Comment:
Would it be sufficient to append the api key rather then the whole config? 

It's quite late and I'm sleepy so do say if I'm missing something!

Reply:
yes, my bad 👍